tests/queries/0_stateless/03174_least_greatest_ignore_null_input_values.reference

```diff
@@ -0,0 +1,21 @@
+Test with one const argument
+\N	\N
+Test with two const arguments
+1	1
+1	1
+1.0	1.0
```

Comment:
@KevinyhZou I changed the float 1.0 to 1.1 in the .sql test and in this .reference file. Since I don't have permission to push directly to your repository (I am getting a "Permission denied error - 403") I could not do the change using a proper text editor on the command line. Instead, I made change in GitHub's web UI. And that messed up tabs and whitespaces in the expected result file, sorry! (CI will complain)

Would you please run the test locally, re-generate the expected results and push once more? Thanks.